onebusaway-android/src/main/java/org/onebusaway/android/tripservice/NotifierTask.java

```diff
@@ -161,23 +165,34 @@ private Notification createNotification(String routeId,
                 .setContentIntent(contentIntent)
                 .setDeleteIntent(deleteIntent)
                 .setContentTitle(title)
-                .setContentText(getNotifyText(routeId, timeDiff))
+                .setContentText(getNotifyText(routeId, timeDiff, isArriving))
                 .build();
 
     }
 
-    private String getNotifyText(String routeId, long timeDiff) {
+    private String getNotifyText(String routeId, long timeDiff, boolean isArriving) {
```

Comment:
FYI - for the unit test in `UIUtilTest.testArrivalInfoLabels()`, the response in the code obtained from `"Hillsborough Area Regional Transit_6497"` is actually mocked from this file - `/onebusaway-android/src/androidTest/res/raw/arrivals_and_departures_for_stop_hart_6497.json`.